test/unit/org/apache/cassandra/index/sai/cql/AbstractQueryTester.java

```diff
@@ -58,20 +67,28 @@ public void setup() throws Throwable
         executor = new SingleNodeExecutor(this, INDEX_QUERY_COUNTER);
     }
 
+    @After
+    public void teardown() throws Throwable
+    {
+        SAIUtil.setLatestVersion(latest);
+    }
+
+
     @SuppressWarnings("unused")
-    @Parameterized.Parameters(name = "{0}")
+    @Parameterized.Parameters(name = "{0}_{1}")
     public static List<Object[]> params() throws Throwable
     {
         List<Object[]> scenarios = new LinkedList<>();
 
-        scenarios.add(new Object[]{ new DataModel.BaseDataModel(DataModel.NORMAL_COLUMNS, DataModel.NORMAL_COLUMN_DATA), IndexQuerySupport.BASE_QUERY_SETS });
-
-        scenarios.add(new Object[]{ new DataModel.CompoundKeyDataModel(DataModel.NORMAL_COLUMNS, DataModel.NORMAL_COLUMN_DATA), IndexQuerySupport.BASE_QUERY_SETS });
-
-        scenarios.add(new Object[]{ new DataModel.CompoundKeyWithStaticsDataModel(DataModel.STATIC_COLUMNS, DataModel.STATIC_COLUMN_DATA), IndexQuerySupport.STATIC_QUERY_SETS });
+        for (Version version : List.of(Version.AA, Version.CA))
```

Comment:
How long does it take to run these tests? I wonder if it is worth just using `Version.ALL` here to prevent missed test updates.

Reply:
Works for me